R/reactives.R

```diff
@@ -357,7 +357,9 @@ Observable <- R6Class(
         stop("Can't make a reactive expression from a function that takes one ",
              "or more parameters; only functions without parameters can be ",
              "reactive.")
-      .func <<- func
+
+      .func <<- wrapFunctionLabel(func, paste("reactive", label),
```

Comment:
Is it necessary to wrap a label here? The name of the reactive is already present in the stack. "count" appears at two levels in this flame graph:

![image](https://cloud.githubusercontent.com/assets/86978/11158758/f58b3e3c-8a1f-11e5-9d39-293bd756edb4.png)

Is it possible to turn on stack tracing between `.func` and the user code, but without the relabeling?